### ISSUE TITLE
4 level deep struct breaks again on the latest commit.

### ISSUE THREAD

**Issue by didip** [cosiner/flag#8]
Works here: https://github.com/cosiner/flag/commit/d92f27aedc79cc2b1cfc52a23b07d9ab408775dc

Stop working here: https://github.com/cosiner/flag/commit/e5854e42cd87a32f1a8a21b006033b66dde8ca60

Example config struct:

```
type Arguments struct {
    Config string `names:"-c" default:"." desc:"Path to config directory"`

    Index struct {
        Enable bool

        Full struct {
            Enable bool

            List struct {
                Enable   bool
                Unsynced bool `names:"--unsynced" default:"false" desc:"Display only unsynced metric indices"`
            }

            Syncto struct {
                Enable bool

                Storage struct {
                    Enable   bool
                    Unsynced bool `names:"--unsynced" default:"false" desc:"Sync only unsynced metric indices"`
                }
            }
        }

        Short struct {
            Enable bool

            List struct {
                Enable   bool
                Unsynced bool `names:"--unsynced" default:"false" desc:"Display only unsynced metric indices"`
            }

            Syncto struct {
                Enable bool

                Storage struct {
                    Enable   bool
                    Unsynced bool `names:"--unsynced" default:"false" desc:"Sync only unsynced metric indices"`
                }
            }
        }
    }
}
```

Error:

```
go run main.go index short syncto storage
StandaloneValue: standalone value without flag: [main index short syncto] storage
```

**Comment by cosiner** [comment]
@didip Sorry, i'm so stupid to just fixed only one variable name in #7 , it exactly has two names need to be fixed. You can take a look of the latest commit https://github.com/cosiner/flag/commit/e5e06a5be4a39c3d65d0986890d13158b784fb0f.

**Comment by didip** [comment]
It's no problem at all :) Thank you for fixing it very quickly.

I have confirmed that the latest commit works.
